keystrokes/src/components/Character/CharacterCreationModal.js

```diff
@@ -0,0 +1,118 @@
+import React, {useState} from 'react';
+import {Modal, Button, TextField, FormControl, InputLabel, Select, MenuItem, Grid} from '@material-ui/core';
+import { characterDetails } from '../../config/constants';
+import styleGuide from '../../stylesheets/style';
+
+function CharacterCreationModal({handleSubmit, isOpen, setIsOpen}) {
```

Comment:
Nice use of deconstruction here, but i don't see prop types defined. eslint might help remind to do this.

Reply:
good point, added eslint to the project, not sure how to do an indepth configuration but the rc file is there atleast.